docs/reference/conditional_validation.rst

```diff
@@ -116,13 +109,11 @@ The method must accept two arguments:
  - ``ErrorElement``: the instance where assertion can be checked
  - ``value``: the object instance
 
-Example from the ``SonataPageBundle``
```

Comment:
I think that `the` should be here? cc @greg0ire

Reply:
I think it should be either "Example from the Sonata page bundle" or "Example from `SonataPageBundle`"